src/login/login.resource.tsx

```diff
@@ -1,7 +1,7 @@
 export function performLogin(username, password) {
   const token = window.btoa(`${username}:${password}`);
   return window
-    .fetch(`/openmrs/ws/rest/v1/session`, {
+    .fetch(window.getOpenmrsSpaBase() + `/ws/rest/v1/session`, {
```

Comment:
This should actually be `window.openmrsBase`, not `window.getOpenmrsSpaBase()`.

Reason is that we want to make requests to `/openmrs/ws/rest/v1/session`, not to `/openmrs/spa/ws/rest/v1/session`. See https://github.com/openmrs/openmrs-module-spa/blob/master/omod/src/main/webapp/master-single-page-application.jsp#L20-L22.

This is something that will be covered by `openmrsFetch`